signac/common/crypt.py

```diff
@@ -28,15 +29,13 @@ def get_keyring():
         "Return the system user keyring."
         return keyring.get_keyring()
 
-# this is here because of issues importing the same variable in
-# signac/__init__.py from the top level namespace
-__version__ = '1.2.0'
-
 
 # this class is deprecated
 class SimpleKeyring(object):
     """Simple in-memory keyring for caching."""
 
+    @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+                details="The crypt module is deprecated.")
```

Comment:
Is this class sub-classed? Otherwise, I'd prefer to apply the decorator the class whenever possible.

Reply:
I am just keeping the deprecation method consistent between classes which are subclassed and classes which aren't. Although I don't really have a preference for one or the other.

Reply:
Please deprecate the class directly if you're not prohibited from doing so by inheritance.